packages/grafana-ui/src/components/uPlot/Plot.tsx

```diff
@@ -81,8 +88,17 @@ export const UPlotChart: React.FC<PlotProps> = props => {
 
   // Memoize plot context
   const plotCtx = useMemo(() => {
-    return buildPlotContext(registerPlugin, addSeries, addAxis, addScale, canvasRef, props.data, plotInstance);
-  }, [registerPlugin, addSeries, addAxis, addScale, canvasRef, props.data, plotInstance]);
+    return buildPlotContext(
+      !!plotInstance,
```

Comment:
nit: maybe use Boolean
```suggestion
      Boolean(plotInstance),
```